src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -293,7 +308,7 @@ const artworkFilterReducer = (
         "inquireableOnly",
         "offerable",
       ]
-      booleanFilterTypes.forEach(filter => {
+      booleanFilterTypes.forEach((filter) => {
```

Comment:
If you rebase from master and run prettier over these files the churn has been fixed.